crates/pathfinder/src/storage.rs

```diff
@@ -211,40 +230,89 @@ pub(crate) mod test_utils {
     }
 
     /// Creates a set of test transactions and receipts.
-    pub(crate) fn create_transactions_and_receipts<const N: usize>(
-    ) -> [(transaction::Transaction, transaction::Receipt); N] {
-        let transactions = (0..N).map(|i| transaction::Transaction {
-            calldata: None,
-            class_hash: None,
-            constructor_calldata: None,
-            contract_address: Some(ContractAddress(
-                StarkHash::from_hex_str(&"2".repeat(i + 3)).unwrap(),
-            )),
-            contract_address_salt: None,
-            entry_point_type: None,
-            entry_point_selector: None,
-            signature: None,
-            transaction_hash: StarknetTransactionHash(
-                StarkHash::from_hex_str(&"f".repeat(i + 3)).unwrap(),
-            ),
-            max_fee: None,
-            nonce: None,
-            r#type: transaction::Type::InvokeFunction,
-            sender_address: None,
-            version: None,
-        });
-        let receipts = (0..N).map(|i| transaction::Receipt {
-            actual_fee: None,
-            events: vec![transaction::Event {
-                from_address: ContractAddress(StarkHash::from_hex_str(&"2".repeat(i + 3)).unwrap()),
-                data: vec![EventData(
+    pub(crate) fn create_transactions_and_receipts(
+    ) -> [(transaction::Transaction, transaction::Receipt); NUM_TRANSACTIONS] {
```

Comment:
Surely this bracket split on line can't be correct format?

Reply:
I'm using rustfmt with default config and I'm afraid that that's what I get :open_mouth: .

Reply:
What's the issue? Looks like all other variations would go over some max-line-width...

Reply:
No issue just struck me as weird :D Would maybe have something more like:
```rust
    pub(crate) fn create_transactions_and_receipts()
        -> [(transaction::Transaction, transaction::Receipt); NUM_TRANSACTIONS] {
```